src/app/filecoin-plus/allocators/data/allocatorsTableColumnsData.ts

```diff
@@ -0,0 +1,54 @@
+import { createColumnHelper } from '@tanstack/react-table'
+
+import { ApplyLink } from '../components/ApplyLink'
+import type { Allocator } from '../schemas/AllocatorSchema'
+
+const columnHelper = createColumnHelper<Allocator>()
+
+export const allocatorsTableColumnsData = [
+  columnHelper.accessor('name', {
+    header: 'Organization Name',
+    cell: (info) => info.getValue(),
+    meta: {
+      headerCellStyle: 'w-72 sm:w-80 lg:w-96',
+      bodyCellStyle: 'truncate',
+    },
+  }),
+  columnHelper.accessor('metapathway_type', {
+    header: 'Type',
+    cell: (info) => info.getValue(),
+    meta: {
+      headerCellStyle: 'w-44',
+    },
+  }),
+  columnHelper.accessor('location', {
+    header: 'Region',
+    cell: (info) => info.getValue(),
+    meta: {
+      headerCellStyle: 'w-44',
+    },
+  }),
+  columnHelper.accessor('application.required_replicas', {
+    header: 'Required Replicas',
+    cell: (info) => info.getValue(),
+    meta: {
+      headerCellStyle: 'w-44',
+    },
+  }),
+  columnHelper.accessor('application.required_sps', {
+    header: 'Required SPs',
+    cell: (info) => info.getValue(),
+    meta: {
+      headerCellStyle: 'w-44',
+    },
+  }),
+  columnHelper.accessor('application.allocation_bookkeeping', {
+    header: 'Apply',
+    meta: {
+      headerCellStyle: 'w-32',
+      bodyCellStyle: 'text-brand-300',
+    },
+    enableGlobalFilter: false,
+    cell: ApplyLink,
```

Comment:
Same comment [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/743/files#r1812345674)

Reply:
@mirhamasala I'm confused, I thought we agreed on this pattern - `cell: ApplyLink` - in a [comment](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/743#discussion_r1810967008) above?

I can definitely update it, but then we will be using a UI component in the data table, which you didn't like?

Let me know :)